### DISCUSSION TITLE
crash at attemt to supply ioproxy to format that does not support it.

### DISCUSSION THREAD

**Discussion by geniusisme** [OpenImageIO/oiio#3451] [General]
https://github.com/OpenImageIO/oiio/blob/1e74b7ecf54b9b8c851f91cf3a99812f863ad150/src/libOpenImageIO/imageioplugin.cpp#L553

Pointer is reset, and then out->format_name() is called on it. Should I file an issue?

**Comment by lgritz** [comment]
I think we can directly turn this into an issue!
Thanks, you are quite right about the problem. I will submit a fix immediately.

**Reply by lgritz** [reply]
Proposed fix in #3453

**Reply by geniusisme** [reply]
Thank you.

**Reply by lgritz** [reply]
I've pushed the fix to master as well as dev-2.3, and it will be included in the next 2.3.17.0 release scheduled for this coming week.

I'm going to close this discussion since it was transferred to an issue.
